principles.md

```diff
@@ -0,0 +1,20 @@
+## Product and Engineering Principles
+
+**We believe in being paid for quality work**
+
+There's an incredible amount of software infrastructure that is under funded and ignored.
+We believe our software needs to generate revenue to employ the best minds.
+
+**We balance openness and transparency**
+
+We work as much as we can in the open; our commits, designs, and comments are largely in public repositories for anyone to use and observe.
+We strive to find balance in open-core and community friendly software versus competing in a dynamic and evolving market segment.
```

Comment:
I don't understand this sentence.
Is this trying to say "We build an open-core product focused on our users but also don't ignore our competition"?

Reply:
Not exactly. When I jotted this down I think what I was trying to convey is we're a commercial for profit company that attempts to balance that goal with also producing useful open and useful software for the community, and that this is a spectrum that needs to be balanced.

Reply:
Gotcha. How about making this more direct, like:
`We balance between being a for-profit company and a group of engineers producing open, free and useful software.`